assembly/vm-mock/storage.ts

```diff
@@ -10,7 +12,7 @@
  */
 // @ts-ignore: decorator
 @external("massa-sc-std", "set")
```

Comment:
massa-sc-std is deprecated, what is the purpose of this annotation @external

Reply:
ok, this file can be deleted then

Reply:
Only setData seems to be not used, but others are used for example here: https://github.com/massalabs/massa-sc-contracts/blob/main/assembly/erc20/__test__/erc20-mint.spec.ts#L14